### PR TITLE
Add TwoFun game for touch devices

### PR DESCRIPTION
**Pull request by janbrummer**
Simple touch based game for the Librem 5 phone.

### CODE REVIEW

File: org.tabos.Twofun.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "modules": [
+        {
+            "name": "twofun",
+            "buildsystem": "meson",
+            "config-opts": [ "--libdir=lib" ],
+            "builddir": true,
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.com/tabos/twofun.git"
```
Comment (nedrichards) [inline org.tabos.Twofun.json]: you're missing a comma here.

**Comment by LeandroStanger** [comment]
bot, build org.tabos.Twofun

File: org.tabos.Twofun.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland"
+    ],
+    "modules": [
+        {
+            "name": "twofun",
+            "buildsystem": "meson",
+            "config-opts": [ "--libdir=lib" ],
+            "builddir": true,
```
Comment (bilelmoussaoui) [inline org.tabos.Twofun.json]:
```suggestion
```
already set by default.

File: org.tabos.Twofun.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland"
+    ],
+    "modules": [
+        {
+            "name": "twofun",
+            "buildsystem": "meson",
+            "config-opts": [ "--libdir=lib" ],
```
Comment (bilelmoussaoui) [inline org.tabos.Twofun.json]:
```suggestion

```

already set by default.

**Comment by bilelmoussaoui** [comment]
bot, build org.tabos.Twofun

**Comment by bilelmoussaoui** [comment]
OARS https://hughsie.github.io/oars/generate.html and release tag are missing from the appdata file.

**Comment by bilelmoussaoui** [comment]
bot, build org.tabos.Twofun

**Comment by bilelmoussaoui** [comment]
Thanks! Everything works fine. Do you own tabos.org?

**Comment by janbrummer** [comment]
Yes, i do.

**Comment by bilelmoussaoui** [comment]
Repository has been created: https://github.com/flathub/org.tabos.Twofun


Thanks!

**Pull request closed without merging**